.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release auto update
+
+on:
+  pull_request:
+    types:
+    - closed
+    branches: 
+    - main
+
+jobs:
+  release:
+    name: auto_release
+    if: github.event.pull_request.merged == true
+    uses: mondu-ai/release-action/.github/workflows/release-update.yml@master
+    secrets: inherit
+    with:
+      asset_creation_script: |
+        echo "FILES=""" >> $GITHUB_ENV
+
+      post_script: |
+        UPDATE_URL="https://packagist.org/api/update-package?username=MonduPlugins&apiToken=${{ secrets.API_TOKEN }}"
```

Comment:
**nitpick(non-blocking)**: it would be great to change `secrets.API_TOKEN` to something more specific. For example `secrets.MAGENTO_2_PACKAGIST_API_TOKEN`. It would be much easy to maintain in the future

Reply:
Yes, sure